partner_tag_smart_assignation/models/hr_department.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
```

Comment:
Your module could be useful for many organisations that do not have `hr`  installed. So better not to have `hr`  as a dependency. Put the `hr` modification in a separate module (partner_tag_smart_assignation_hr), that depends on both this module and hr.